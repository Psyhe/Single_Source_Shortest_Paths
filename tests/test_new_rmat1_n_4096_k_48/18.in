4096 1546 1630
1546 148 217
1546 2592 244
1546 1032 165
1546 24 54
1546 2224 54
1546 2064 130
1546 1032 161
1546 1344 224
1546 248 6
1546 2218 191
1546 2056 113
1546 2113 9
1546 337 62
1546 3781 231
1546 257 150
1546 101 40
1546 1 228
1546 32 57
1546 0 68
1546 2308 158
1546 2152 121
1546 279 77
1546 1864 96
1547 52 113
1547 2114 1
1547 2121 25
1547 129 187
1547 48 198
1547 725 11
1547 256 34
1547 132 116
1547 1029 125
1548 514 10
1548 160 154
1548 0 200
1548 72 247
1548 2144 191
1548 32 51
1548 194 205
1548 35 69
1548 2048 162
1548 2097 2
1548 72 183
1548 385 148
1548 260 82
1548 3105 83
1548 69 221
1548 2056 187
1548 516 66
1548 2056 232
1548 1856 22
1548 274 151
1548 3626 93
1548 2480 85
1548 57 33
1548 1664 81
1548 1123 215
1548 129 109
1548 1664 8
1548 37 195
1549 2577 76
1549 64 141
1549 386 225
1549 514 154
1549 815 80
1549 640 172
1549 2188 34
1550 2 122
1550 43 27
1550 2564 118
1550 0 66
1550 800 181
1550 400 151
1550 872 229
1550 1090 14
1550 1792 171
1551 0 126
1551 145 9
1551 1056 37
1552 104 184
1552 2056 21
1552 299 68
1552 2528 103
1552 16 81
1552 442 167
1552 1283 59
1552 585 46
1552 2562 243
1552 2713 38
1552 3424 240
1552 2170 156
1552 270 121
1552 3092 140
1552 2048 55
1552 1288 103
1552 2144 94
1552 2182 163
1552 537 181
1552 83 125
1552 1088 28
1552 0 41
1552 571 101
1552 2049 31
1552 400 170
1552 544 2
1552 773 188
1552 3584 147
1552 12 129
1552 2068 185
1552 20 33
1552 194 26
1552 2306 171
1552 2 80
1552 90 85
1552 260 146
1552 416 208
1552 2368 151
1552 1239 11
1552 2372 170
1552 2816 105
1552 1669 57
1552 2472 88
1552 3103 12
1552 8 137
1552 516 224
1552 68 176
1552 3348 158
1552 2051 78
1552 2340 104
1552 2178 44
1552 2081 5
1552 576 192
1552 2560 45
1552 0 140
1552 135 252
1552 193 23
1552 138 3
1552 1024 232
1552 45 193
1552 2 175
1552 21 95
1552 1098 108
1553 1092 113
1553 3976 176
1553 258 1
1553 1030 73
1553 192 80
1553 1576 3
1553 2112 171
1553 1028 232
1553 1056 194
1553 706 136
1553 4 171
1553 2560 41
1553 80 240
1553 2544 252
1553 4 249
1553 1107 44
1553 12 131
1553 2061 198
1553 384 143
1553 1024 78
1553 128 192
1553 262 198
1553 2130 210
1554 2048 99
1554 96 79
1554 1346 211
1554 520 147
1554 88 143
1554 1 180
1554 2118 188
1554 2080 221
1554 3080 235
1554 524 47
1554 1476 83
1554 3340 78
1554 38 142
1554 897 110
1554 9 213
1554 272 115
1554 257 206
1554 2744 72
1554 161 17
1554 72 93
1554 272 238
1554 32 194
1554 1 109
1554 64 133
1554 34 204
1554 579 96
1554 2196 212
1554 44 88
1554 2064 250
1555 1408 230
1555 32 98
1555 361 178
1555 137 64
1555 1383 247
1555 0 202
1555 9 79
1555 1034 114
1556 3074 63
1556 514 114
1556 2786 64
1556 3329 125
1556 2112 149
1556 1472 50
1556 68 24
1556 0 57
1556 1931 255
1556 3393 164
1556 2465 193
1556 2956 174
1556 3094 122
1556 1162 136
1556 8 43
1556 640 160
1556 1408 87
1556 777 11
1556 576 82
1557 9 46
1557 410 73
1558 384 115
1558 916 151
1558 344 5
1558 516 247
1558 418 46
1558 20 98
1559 528 6
1559 570 38
1560 173 80
1560 1304 0
1560 1024 150
1560 0 128
1560 232 193
1560 1284 34
1560 1188 90
1560 2194 115
1560 3105 157
1560 98 60
1560 1192 25
1560 654 111
1560 641 69
1560 3216 46
1560 2306 150
1560 2849 0
1560 128 206
1560 32 112
1561 130 155
1561 2127 130
1561 3072 102
1561 37 157
1561 4 153
1561 2305 232
1561 389 52
1562 512 105
1562 3 246
1562 392 215
1564 21 97
1564 1 124
1564 545 28
1564 4 163
1564 12 201
1564 16 149
1564 1160 42
1564 518 35
1564 2050 2
1564 512 106
1564 2 167
1565 96 247
1565 2336 202
1566 131 118
1568 260 67
1568 1040 140
1568 166 141
1568 1280 58
1568 914 32
1568 0 227
1568 72 103
1568 2176 97
1568 14 190
1568 160 182
1568 784 212
1568 1156 200
1568 1 148
1568 2061 136
1568 0 171
1568 2196 20
1568 0 175
1568 1066 0
1568 904 44
1568 528 5
1568 2314 36
1568 608 72
1568 2560 96
1568 69 152
1568 2197 134
1568 258 185
1568 738 104
1568 26 120
1568 148 59
1568 320 36
1568 692 97
1568 3080 167
1568 20 49
1568 0 13
1568 2116 96
1568 718 50
1568 3 191
1568 64 43
1568 8 35
1568 258 80
1568 276 151
1568 1026 90
1568 452 117
1568 1048 81
1568 16 88
1568 4 106
1568 10 4
1568 2112 135
1568 0 128
1568 678 245
1568 32 255
1568 786 126
1568 384 79
1568 288 28
1568 532 205
1568 193 68
1568 3 188
1568 512 69
1568 170 35
1568 832 30
1568 676 82
1568 658 96
1568 4 38
1568 2704 219
1568 8 0
1568 548 129
1568 6 88
1568 419 121
1568 386 120
1568 1 163
1568 17 241
1568 512 143
1568 0 109
1568 80 253
1568 1 94
1569 16 244
1569 64 82
1569 2580 74
1569 2128 223
1569 2058 125
1569 1328 234
1569 832 93
1569 0 98
1569 2196 159
1569 1044 163
1569 1024 215
1569 28 232
1569 512 69
1569 3938 175
1569 2048 228
1569 1610 157
1569 18 216
1569 2304 147
1569 1156 172
1569 33 195
1569 2049 153
1569 464 64
1570 3145 252
1570 256 126
1570 523 43
1570 34 19
1570 2114 196
1570 2336 77
1570 3328 7
1570 2777 97
1570 420 23
1570 20 82
1570 1 166
1570 523 175
1570 224 2
1570 2817 156
1570 257 223
1570 88 239
1570 324 14
1570 1024 118
1570 2376 123
1570 2144 46
1570 640 143
1570 3344 14
1570 273 241
1570 512 178
1570 4 173
1571 516 143
1571 2704 44
1571 790 32
1571 280 50
1571 2072 222
1571 136 32
1571 0 174
1571 610 54
1571 941 180
1572 68 242
1572 1024 25
1572 97 176
1572 40 20
1572 1048 54
1572 72 50
1572 1024 241
1572 2112 117
1572 4 218
1572 391 71
1572 12 27
1572 2049 116
1572 2160 122
1572 514 31
1572 2186 185
1572 1024 109
1572 520 15
1572 12 20
1572 2112 245
1572 2318 73
1572 65 78
1573 3537 242
1573 1128 15
1573 18 139
1573 8 152
1573 64 205
1573 1024 77
1573 2200 67
1573 580 221
1574 8 172
1574 2048 100
1574 2065 44
1574 2121 87
1574 231 10
1574 616 94
1575 1094 207
1575 450 97
1575 72 242
1575 2817 228
1575 18 56
1576 1189 127
1576 177 245
1576 2435 110
1576 0 56
1576 1088 78
1576 2356 222
1576 4 53
1576 516 200
1576 1049 126
1576 2065 196
1576 256 244
1576 1 114
1576 165 37
1576 2371 21
1576 513 192
1576 2112 108
1576 1057 181
1576 288 127
1576 2049 48
1576 165 205
1576 386 50
1576 400 64
1576 288 201
1576 74 65
1576 1553 3
1576 2179 44
1576 1040 255
1576 4 178
1576 1219 137
1576 2448 144
1577 204 33
1577 68 250
1577 554 82
1577 2896 76
1578 2084 169
1578 2304 86
1578 2800 71
1578 2177 37
1578 2064 30
1578 3018 195
1578 2048 153
1578 2432 38
1579 512 100
1580 132 135
1580 128 49
1580 130 163
1580 1089 183
1580 2896 18
1580 12 14
1580 576 6
1580 3626 61
1580 1024 6
1580 66 39
1581 3216 38
1581 304 183
1582 90 65
1583 256 27
1584 836 52
1584 2324 135
1584 40 102
1584 0 219
1584 1024 201
1584 37 146
1584 32 168
1584 2080 93
1584 2125 7
1584 1 254
1584 3075 173
1584 261 225
1584 2944 54
1584 288 19
1584 2625 210
1584 2 48
1584 2153 75
1584 448 68
1584 2066 165
1584 256 252
1584 2064 237
1584 0 55
1584 2704 174
1584 2707 192
1584 528 146
1584 216 183
1584 1312 143
1585 2114 146
1585 21 96
1585 0 14
1585 2192 30
1585 76 215
1585 140 70
1585 661 254
1586 1056 250
1586 2304 194
1586 1024 186
1586 32 161
1586 512 83
1586 3108 82
1586 668 138
1586 2629 5
1586 64 37
1586 1096 120
1587 1296 149
1587 272 155
1587 24 51
1587 2816 233
1588 49 39
1588 265 203
1588 520 227
1589 2114 35
1590 11 147
1591 68 148
1592 460 98
1592 144 130
1592 162 202
1592 1348 143
1592 2070 52
1593 263 10
1593 256 9
1593 520 18
1594 409 6
1594 296 175
1594 356 22
1596 2 15
1596 8 101
1596 96 233
1597 3584 124
1599 720 219
1600 1280 76
1600 2056 4
1600 300 0
1600 2060 204
1600 258 181
1600 2069 2
1600 1112 16
1600 3 207
1600 648 0
1600 1037 41
1600 9 201
1600 35 142
1600 262 253
1600 1284 79
1600 6 164
1600 1040 77
1600 48 154
1600 162 161
1600 4 224
1600 2176 128
1600 1282 119
1600 1057 139
1600 9 198
1600 1227 176
1600 1360 74
1600 0 166
1600 8 113
1600 544 178
1600 2125 208
1600 128 124
1600 418 76
1600 289 54
1600 262 112
1600 384 185
1600 2324 126
1600 64 115
1600 1 4
1600 1 138
1600 33 211
1600 1208 248
1600 289 180
1600 1300 79
1600 1297 249
1600 2330 230
1600 256 37
1600 1706 66
1600 1090 227
1600 97 237
1600 1475 46
1600 129 3
1600 3072 199
1600 2 192
1600 64 27
1600 1096 112
1600 2332 212
1600 1281 83
1600 2050 239
1600 2176 138
1600 2064 88
1600 26 54
1600 0 152
1600 276 105
1600 2304 100
1600 3584 159
1600 1413 133
1600 256 193
1600 8 225
1600 11 149
1601 1217 62
1601 816 100
1601 1357 107
1601 34 105
1601 515 32
1601 36 9
1601 772 211
1601 577 77
1601 98 17
1601 275 17
1601 1664 63
1601 2432 181
1601 1152 92
1601 2562 112
1601 48 74
1601 2433 167
1601 2216 81
1601 2052 248
1601 32 196
1602 1539 206
1602 1208 140
1602 1418 9
1602 1121 49
1602 9 184
1602 32 144
1602 0 55
1602 581 193
1602 2089 26
1602 1 169
1602 256 64
1602 516 183
1602 2054 150
1602 1166 123
1602 136 127
1602 578 112
1602 35 151
1602 2066 173
1602 264 8
1602 274 147
1602 1034 38
1602 2346 52
1602 796 93
1602 3479 219
1602 1152 251
1602 32 90
1602 993 229
1602 0 188
1602 352 136
1602 456 248
1602 545 157
1603 3464 254
1603 136 146
1603 161 8
1603 2 235
1603 2571 206
1603 329 163
1603 2083 182
1603 288 216
1604 57 141
1604 0 150
1604 96 108
1604 256 242
1604 1282 101
1604 336 43
1604 2124 201
1604 2113 156
1604 516 215
1604 1034 16
1604 1092 10
1604 2016 102
1604 400 163
1604 3128 34
1604 544 199
1604 320 70
1604 1 101
1604 3349 134
1604 32 5
1605 1424 252
1605 2176 191
1605 2080 61
1605 0 178
1605 513 109
1605 1160 229
1606 660 244
1606 64 254
1606 2403 169
1606 2656 202
1608 1096 209
1608 19 119
1608 1297 128
1608 2129 56
1608 1282 194
1608 1024 116
1608 872 154
1608 324 203
1608 1041 7
1608 16 188
1608 256 90
1608 128 23
1608 1052 176
1608 2625 30
1608 873 175
1608 1168 177
1608 537 158
1608 2054 187
1608 736 121
1609 514 184
1609 4 127
1609 32 155
1609 2192 134
1609 773 155
1609 281 132
1609 513 84
1610 2054 13
1610 1 62
1610 1569 157
1610 520 245
1610 160 203
1610 132 66
1610 9 6
1611 228 196
1611 11 193
1611 560 124
1611 256 185
1611 128 50
1612 0 79
1612 1314 250
1612 2 208
1612 2056 54
1612 72 18
1612 2052 167
1612 836 74
1613 1092 219
1613 168 187
1614 128 94
1614 1041 199
1614 0 215
1614 308 77
1614 2328 109
1614 898 46
1615 59 172
1615 274 246
1616 10 109
1616 1 165
1616 516 138
1616 8 112
1616 2352 87
1616 2184 23
1616 8 7
1616 2055 109
1616 818 111
1616 449 2
1616 513 154
1616 260 17
1616 492 62
1616 128 59
1616 420 94
1616 1200 68
1616 8 8
1616 976 82
1616 1 163
1616 2220 161
1616 705 252
1616 151 26
1616 5 211
1616 128 239
1616 3119 141
1616 1044 93
1616 546 194
1616 2304 242
1616 32 188
1616 2049 162
1616 48 67
1616 41 27
1616 1024 43
1616 1322 33
1617 68 24
1617 3080 12
1617 448 79
1617 584 52
1617 1284 185
1617 2048 42
1617 768 21
1617 388 211
1618 592 16
1618 128 236
1618 1152 79
1618 228 127
1618 400 218
1618 2436 249
1618 10 78
1618 128 177
1618 8 126
1620 2048 18
1620 580 61
1620 8 67
1620 3200 125
1620 160 74
1620 104 114
1620 640 133
1620 2064 173
1621 2744 90
1621 0 95
1621 258 252
1621 3096 193
1622 2088 169
1622 1953 40
1623 264 35
1624 720 60
1624 2840 11
1624 3284 144
1624 0 67
1625 32 23
1627 2080 172
1628 4 29
