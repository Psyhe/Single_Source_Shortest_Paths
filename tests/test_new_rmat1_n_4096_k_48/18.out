15
3
14
41
21
17
4
6
26
68
20
54
13
6
25
65
106
1000000000000000000
5
207
134
1000000000000000000
2
70
6
35
17
18
43
62
9
37
32
101
8
46
76
28
22
14
26
53
57
37
151
153
68
10
29
1000000000000000000
16
130
1000000000000000000
223
6
15
14
17
9
69
210
1000000000000000000
10
91
14
50
19
191
50
231
9
18
21
1000000000000000000
19
95
92
41
22
27
1000000000000000000
180
29
1000000000000000000
1000000000000000000
